frontend/app/components/redesign/components/revshare/ShareInput.tsx

```diff
@@ -177,11 +176,8 @@ export const ShareInput = React.memo(
             onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
               onChangeName(e.target.value)
             }
-            aria-describedby={`name-description-${index}`}
+            ariaDescription="Optional name for this recipient."
           />
```

Comment:
Consider using `aria-describedby` instead of inline descriptions, as `*by` ones get translated by browsers.